app/views/podcast_planner/_form_side.html.erb

```diff
@@ -2,47 +2,55 @@
   <div class="card-header-primary">
     <h5 class="card-title"><%= t(".header.preview") %></h5>
   </div>
-  <%= turbo_frame_tag "preview", target: "_top" do %>
-    <% if @planner.dates.present? %>
```

Comment:
also removed this conditional so that the calendars don't depend on the calculator/generator to render.